modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -11,9 +11,9 @@ Consider the following when you implement a Scheduler in your Mule application:
 
 * Schedulers use the same timezone as the machine on which Mule is running. However, if an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic region in which the application is running.
 // TODO: MULE-14438
-* If xref:execution-engine.adoc#backpressure[back-pressure] is triggered because no resources are available at the time of the trigger, that trigger is skipped.
+* If xref:execution-engine.adoc#backpressure[back-pressure] is triggered because no resources are available at the time of the trigger, that trigger is skipped. In this case a *MULE:FLOW_BACK_PRESSURE* error is thrown indicating the flow cannot accept new events at this time.  
```

Comment:
```suggestion
* If xref:execution-engine.adoc#backpressure[back-pressure] is triggered because no resources are available at the time of the trigger, the application skips the trigger and throws a MULE:FLOW_BACK_PRESSURE error. The error indicates that the flow cannot accept new events at that time.  
```

Reply:
moved change to new PR: https://github.com/mulesoft/docs-mule-runtime/pull/1650